generics/services/users.js

```diff
@@ -57,6 +60,228 @@ const profile = function ( token,userId = "" ) {
     })
 }
 
+/**
+  * 
+  * @function
+  * @name learnerLocationSearch
+  * @param {String} bearerToken - autherization token.
+  * @param {object} filterData -  bodydata .
+  * @returns {Promise} returns a promise.
+*/
+
+const learnerLocationSearch = function ( filterData, pageSize = "", pageNo = "", searchKey = "" ) {
```

Comment:
@aks30 both call have difference in body data. Also API returns data in different structure in both cases. It will be nice if we can keep it like this